packages/react-components/src/button/src/Button.jsx

```diff
@@ -125,11 +125,11 @@ export function InnerButton(props) {
         },
         icon: {
             size: condensed ? size : embeddedIconSize(size),
-            className: "o-ui-button-left-icon"
+            className: "o-ui-button-icon o-ui-button-left-icon"
```

Comment:
This modification is wrong.

This is the Button component not the IconButton component. The original code is correct it should be:

```
icon: {
            size: condensed ? size : embeddedIconSize(size),
            className: "o-ui-button-left-icon"
}
```

Reply:
I think we talked about it and it was alright, since `button-icon` implies it's the icon in the button.

Reply:
Yes you're right